pkg/app/app.go

```diff
@@ -582,7 +582,7 @@ func (a *App) PrintState(c StateConfigProvider) error {
 			errs = append(errs, err)
 		}
 
-		return
```

Comment:
Ohh is this unallowed in Go 1.21 now @yxxhero ?

Reply:
no. because the nakedret, linter of golint-ci. shall we disable the nakedret linter? @mumoshu WDYT?

Reply:
Awesome @yxxhero! I'm new to that code review guideline and the nakedret linter. Makes sense. And keep this for now and revising the appropriate setting for nakedret sounds good to me!